Library/Formula/apache-forrest.rb

```diff
@@ -4,6 +4,8 @@ class ApacheForrest < Formula
   url "https://www.apache.org/dyn/closer.cgi?path=forrest/apache-forrest-0.9-sources.tar.gz"
   sha256 "c6ac758db2eb0d4d91bd1733bbbc2dec4fdb33603895c464bcb47a34490fb64d"
 
+  depends_on :java
```

Comment:
Java 1.5; they're pretty basic. Given that (AFAIK) nothing pre-1.6 is available on OS X, do we need to note that?